### ISSUE TITLE
Additional field to verify password

### ISSUE THREAD

**Issue by stonedonkey** [ctm/mb2-doc#142]
Consider additional field to verify password when registering username or email. 
Obviously this goes to bottom of pile of things to worry about.

**Comment by ctm** [comment]
I'm adding this to Better Explanations, because I think this will help out newbies, especially non-BARGErs who may, in the not too distant future, be encouraged to join us.

The obvious place (to me, at least) to ask for verification is in the relatively new confirmation page that I added for #186.

**Comment by ctm** [comment]
I started on this yesterday and got slightly side-tracked by my realization that I know a lot more about certain techniques in Rust, Yew and wasm-bindgen. So, instead of hacking in another bit of modal-dialog creation, revelation, hiding code, I'm taking the time to do it "right" (#794).

Assuming I don't go overboard with the cleanup or get distracted by something else, I should have this feature deployed by this evening's tournament.

**Comment by ctm** [comment]
I did not get the new code ready for this evening's game. I spent hours futzing with `wasm-bindgen` and then made slow and steady progress until I realized my modal dialog had ugly spacing. Took me a while to figure out (or re-remember) that whitespace matters in HTML files, but gets thrown away in yew `html!` invocations.

In theory, I don't have much more to do, and I definitely like the new code a lot more than what I've done in the past.
